tuhi/gui/drawing.py

```diff
@@ -41,10 +43,12 @@ def __init__(self, json_data, zoom, *args, **kwargs):
         self.orientation = Config().orientation
         Config().connect('notify::orientation', self._on_orientation_changed)
         DATA_PATH.mkdir(parents=True, exist_ok=True)
+        PNG_DATA_PATH.mkdir(parents=True, exist_ok=True)
 
         self.json_data = json_data
         self._zoom = zoom
         self.process_svg()  # sets self.svg
+        self.process_png()  # sets self.png
```

Comment:
I don't think this one is needed. we don't use the png in the GUI directly, so we can really just generate those on-demand before export.